manifests/database/postgresql.pp

```diff
@@ -9,12 +9,14 @@
   $manage_server        = $puppetdb::params::manage_dbserver,
   $manage_package_repo  = $puppetdb::params::manage_pg_repo,
   $postgres_version     = $puppetdb::params::postgres_version,
+  $postgres_datadir     = $puppetdb::params::postgres_datadir,
```

Comment:
@petrenkoas83 no worries. If you feel you can find an alternate approach that improves this experience, feel free to raise a PR or whatever :-).